### PR TITLE
Beta Support

### PR DESCRIPTION
**Pull request by dogweather**
- [Trello card](https://trello.com/c/tMZj9EYK/471-support-beta-users)
- Original PR #1322 

(I created this new branch and new PR because I needed to revert code changes.)

### CODE REVIEW

File: .codeclimate.yml [hunk 1]
```diff
@@ -8,6 +8,13 @@ engines:
     enabled: true
   rubocop:
     enabled: true
+    checks:
+      Rubocop/Style/AndOr:
+        enabled: false
+      Rubocop/Style/GuardClause:
+        enabled: false
+      Rubocop/Style/IfUnlessModifier:
+        enabled: false
```
Comment (dogweather) [inline .codeclimate.yml]: Allows us to begin enforcing these locally and allow legacy code to pass at code climate. @phirefly @yozlet @nickbristow @rememberlenny

File: .rubocop.yml [hunk 2]
```diff
@@ -2,32 +2,53 @@ AllCops:
...
+
 Style/SignalException:
   Enabled: false
+
 Style/StringLiterals:
   EnforcedStyle: double_quotes
+
 Style/AndOr:
-  Enabled: false
+  EnforcedStyle: always
```
Comment (dogweather) [inline .rubocop.yml]: Changing to current ruby programming idioms and best practices. Opinions?

File: app/mailers/cancelation_mailer.rb [hunk 3]
```diff
@@ -1,4 +1,6 @@
 class CancelationMailer < ApplicationMailer
+  # TODO: Remove
+  # rubocop:disable Metrics/MethodLength
```
Comment (dogweather) [inline app/mailers/cancelation_mailer.rb]: Manually ignoring certain style violations to allow the online linter to pass.

File: .rubocop.yml (hunk 2 above)
Comment (dogweather) [inline .rubocop.yml]: We had disabled checking for inline if's because of a simplecov failure to count them properly.

**Comment by rememberlenny** [comment]
@dogweather This is one of the most educational PRs I have reviewed. Thank you for taking time to make these updates!

**Comment by dogweather** [comment]
Thanks, @rememberlenny !

**Pull request merged**